app/models/micropost.rb

```diff
@@ -0,0 +1,6 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
```

Comment:
em research dòng code này nhé

Reply:
dòng code này dùng để xác định quan hệ many to one với bảng User, tức là mỗi record của bảng Micropost chỉ thuộc về duy nhất 1 record của bảng User